src/go/rpk/pkg/cli/security/secret/update.go

```diff
@@ -20,15 +21,19 @@ import (
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/publicapi"
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
+	"google.golang.org/genproto/googleapis/rpc/errdetails"
 )
 
 func newUpdateCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	var secretName, secretValue string
+	var scopes []string
 
 	cmd := &cobra.Command{
 		Use:   "update",
 		Short: "Update an existing secret",
-		Long:  "Update an existing secret for your Redpanda Cloud cluster",
+		Long: `Update an existing secret for your Redpanda Cloud cluster.
+
+Scopes define the areas where the secret can be used. Updating a secret will overwrite its scopes. Available scope options are: redpanda_connect, redpanda_cluster`,
```

Comment:
Same here, can we wrap to 80 cols please?

Reply:
sure